crates/mapdiffbot2/src/github_processor.rs

```diff
@@ -1,37 +1,43 @@
-use std::sync::Arc;
-
-use anyhow::{Context, Result};
-use octocrab::models::pulls::FileDiff;
+use diffbot_lib::log::{error, trace};
+use eyre::{Context, Result};
 use octocrab::models::InstallationId;
-use rocket::http::Status;
-use rocket::outcome::Outcome;
-use rocket::request;
-use rocket::request::FromRequest;
-use rocket::tokio::sync::Mutex;
-use rocket::Request;
-use rocket::State;
-
-use crate::CONFIG;
-use diffbot_lib::github::github_api::*;
-use diffbot_lib::github::github_types::*;
-use diffbot_lib::job::types::{Job, JobJournal, JobSender};
+
+use rocket::{
+    http::Status, outcome::Outcome, request, request::FromRequest, tokio::sync::Mutex, Request,
```

Comment:
Should MDB also be using async_mutex instead of rocket::tokio?

Reply:
it can use it's own mutex because it's there

Reply:
hmm, something to do if we yeet rocket, mkay